docs/content/users/configuration/config_yaml.md

```diff
@@ -1,75 +1,74 @@
-| required_docker_compose_version | Specify an alternate docker-compose version for download                      | If set to `v2.8.0`, for example, it will download and use that version instead of the expected version for docker-compose.                                                                                                                                                                                                                                              |
-| use_docker_compose_from_path | Use the system-installed docker-compose                                       | If this is true, then DDEV will use the docker-compose found in on your system's path instead of using its private known-good docker-compose version.                                                                                                                                                                                                                   |
-| no_bind_mounts | Do not use docker bind mounts                                                 | Some docker environments (like remote docker) do not allow bind mounts, so this option turns off those and turns on mutagen and uses volume copies to do what bind mounts would otherwise do.                                                                                                                                                                           |
+| Option                          | Description                                                                   | Notes                                                                                                                                                                                                                                                                                                                                                                   |
+| ------------------------------- | ----------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| nfs_mount_enabled               | Whether to globally enable NFS mounting for all projects                      | Only `true` has any effect. When `true`, NFS will be used on all projects, regardless of any settings in the individual projects.                                                                                                                                                                                                                                       |
```

Comment:
```suggestion
| nfs_mount_enabled               | Whether to globally enable NFS mounting for all projects                      | Only `true` has any effect. When `true`, NFS will be used on all projects, regardless of any settings in the individual projects. This is rarely used because mutagen has such widespread usage and is so much faster.                                                                                                                                                                                                                                      |
```